manage_sections/templates/manage_sections/section_details.html

```diff
@@ -6,17 +6,21 @@
 <link rel="stylesheet" type="text/css" href="{% static "manage_sections/css/manage_sections.css" %}" media="screen"/>
 {% endblock stylesheet %}
 
+{% block breadcrumb %}
+<nav>
+    <h3>
+        <a href="{% url 'dashboard_course' %}">Manage Course</a>&nbsp;<i class="fa fa-chevron-right"></i>
+        <a href="{% url 'manage_sections:create_section_form' %}">&nbsp;Manage Sections</a>&nbsp;<i class="fa fa-chevron-right"></i>
+        <span>{{section_name}} (<span class="enroll_count"></span>)</span>
```

Comment:
For really long ones, we may want to ensure that the line spacing/padding is what UX is happy with.